index.html

```diff
@@ -50,46 +50,36 @@
   <!-- End Matomo Code -->
 </head>
 <body>
-	<nav>
-		<div class="language-menu" itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
-			<span class="newsletter-date">July-October, 2019</span>
-      <a class="language-menu--item" href="/" itemprop='url'>
-        <span itemprop="name">English</span>
-      </a>
-      <a class="language-menu--item" href="/es/" itemprop='url'>
-        <span itemprop="name">Español</span>
-      </a>
-		</div>
-    <div class="primary-nav" itemscope itemtype="http://schema.org/Organization">
-      <a class="logo-btn" href="/" itemprop="url">
+  <header>
+  	<nav itemscope="itemscope" itemtype="http://schema.org/SiteNavigationElement">
+  		<div class="menu">
+        <span class="menu__news-name">MACILE Newsletter</span>
+        <div class="wrapper">
+          <a class="menu__item" href="/" itemprop='url'>
+            <span itemprop="name">EN</span>
+          </a>
+          <a class="menu__item" href="/es/" itemprop='url'>
+            <span itemprop="name">ES</span>
+          </a>
+        </div>
+  		</div>
+      <a class="logo__btn" href="/" itemprop="url">
         <img
-				  srcset="/app/assets/images/logo-macile-167x180.jpg 167w,
-				          /app/assets/images/logo-macile-116x125.jpg 116w,
-				          /app/assets/images/logo-macile-84x90.jpg 84w"
-				  sizes="(min-width: 800px) 167px, (min-width: 500px) 116px, (min-width: 150px) 84px"
-				  alt="logo-macile">
-			</a>
-      <div class="logo-text">
-        <h1 class="logo-brand" itemprop="brand">MACILE</h1>
-        <ul class="logo-description" itemprop="slogan">
-          <li class="logo-description--text">Mathematics</li>
-          <li class="logo-description--text">Science</li>
-          <li class="logo-description--text">Engineering</li>
-          <li class="logo-description--text">Language</li>
-        </ul>
-      </div>
-		</div>
-  </nav>
-	<!-- A <section> acts just like a <div> except it tells browsers and search engines that it is a section on our page -->
-	<section class="home-hero">
-		<div class="home-hero--inner">
-			<h1 class="home-hero--title">Our awesome graduation</h1>
-			<h2 class="home-hero--subtitle">Conquered steps are milestones on the journeys we're pursuing</h2>
-			<a href="#" class="home-hero-btn">Forging ahead...! </a>
-		</div>
-	</section>
-
-	<!--Like <section>, <article> acts very similar to a <div>. It is new in HTML5. It specifies independent, self-contained content. Can be used for forum post, blog post, news story, comment.-->
+          srcset="/app/assets/images/logo-macile-167x180.png 167w,
+                  /app/assets/images/logo-macile-116x125.png 116w,
+                  /app/assets/images/logo-macile-84x90.png 84w"
+          sizes="(min-width: 800px) 167px, (min-width: 500px) 116px, (min-width: 150px) 84px"
+          alt="logo-macile">
+      </a>
+    </nav>
+  	<section class="home-hero">
```

Comment:
A `header` element likely would not have a `section` element inside of it because they header is used to introduce new sectioning elements. 

By moving the nav out of the header we can clean-up and simplify this code to make it more intuitive. Just add the `home-hero` class directly to the `header` element. We get rid of having to have an extra wrapper element and the class is more clearly tied to the html element that represents its function.